com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/ThirdPartyDemand/AdMob/AdMobAndroid.cs

```diff
@@ -70,6 +70,8 @@ public BidRequest ModifyRequest(BidRequest bidRequest, string data) {
                       "fetchAdMobRewardedSignal", args, 500L);
                    break;
              }
+             bidRequest.User ??= new User();
```

Comment:
Note that ??= was introduced in more recent versions of c#. Years ago we have a publisher complain because there app was on a old enough version of C# where this sugar was not supported.

Reply:
The null coalescing assignment operator looks like it was added in C# 8 which was supported in Unity 2020 so we should be good here.